recommendations/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
+from django.contrib.postgres.fields import ArrayField
+from .choices import *
 
 
-# Create your models here.
 class Movie(models.Model):
-    name = models.CharField(max_length=30, unique=False)
+    name = models.CharField(max_length=30)
```

Comment:
Max_length should be increased to maybe 80? 'Dr. Strangelove or: How I Learned to Stop Worrying and Love the Bomb' is 68 characters

Reply:
DUH thank you